.gitignore

```diff
@@ -18,6 +18,7 @@ DerivedData/
 xcuserdata/
 
 ## Other
+*.D_Store
```

Comment:
intentional?

Reply:
This is what I told you about with regards to the git

Reply:
So weird. Makes sense I guess.